src/test/java/org/jitsi/meet/test/web/WebParticipant.java

```diff
@@ -157,6 +159,126 @@ protected void doHangUp()
         MeetUtils.waitForPageToLoad(driver);
     }
 
+    /**
+     * Executes a script in this {@link WebParticipant}'s {@link WebDriver}.
+     * See {@link JavascriptExecutor#executeScript(String, Object...)}.
+     */
+    @Override
+    public Object executeScript(String var1, Object... var2)
+    {
+        JavascriptExecutor executor = getJSExecutor();
+
+        return executor != null
+            ? executor.executeScript(var1, var2) : null;
```

Comment:
I know this is just code being moved, but it looks like it could swallow errors. The "driver" field is final and non-null, so "executor == null" only happens if the driver is not an instance of JavascriptExecutor. If this happens for WebParticipant, we should throw an exception and not silently return null.

Reply:
true - will have this fixed